salt/proxy/junos.py

```diff
@@ -37,6 +37,9 @@
 
 import logging
 
+## DGM
```

Comment:
DMG?

Reply:
Ok, I think I got it.

Lets remove comments on the imports.
Comments inside try/except are ok just to explain what's going on.

Reply:
fixed